mac/setup/build.sh

```diff
@@ -25,7 +27,7 @@ KEYMANAPP=../Keyman4MacIM/build/Release/Keyman.app
 # Build basic app and copy icons
 #
 
-if [ -f "$SOURCEAPP" ]; then
+if [ -e "$SOURCEAPP" ]; then
   rm -rf "$SOURCEAPP"
 fi
```

Comment:
`-f` matches when "FILE" is a plain and simple file, but not when a directory.

Using `-e` - essentially, "is a valid path" or `-d` - "is a directory" will work for this check, while the original does not.